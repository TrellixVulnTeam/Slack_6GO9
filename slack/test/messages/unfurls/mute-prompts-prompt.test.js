const MutePromptsPrompt = require('../../../lib/messages/unfurls/mute-prompts-prompt');

describe('AutoUnfurlPrompt message rendering', () => {
  test('works', async () => {
    const message = new MutePromptsPrompt();
    expect(message.getAttachment()).toMatchSnapshot();
  });
});
