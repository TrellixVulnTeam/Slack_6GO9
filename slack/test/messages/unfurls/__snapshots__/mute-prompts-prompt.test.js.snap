// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AutoUnfurlPrompt message rendering works 1`] = `
Object {
  "actions": Array [
    Object {
      "name": "mute-24h",
      "text": "Mute prompts for 24h",
      "type": "button",
    },
    Object {
      "name": "mute-indefinitely",
      "text": "Mute prompts indefinitely",
      "type": "button",
    },
    Object {
      "name": "cancel",
      "text": "Always ask me",
      "type": "button",
    },
  ],
  "callback_id": "unfurl-mute-prompts",
  "color": "#24292f",
  "title": "Getting too many prompts?",
}
`;
